benchmarks/dynamo/common.py

```diff
@@ -516,6 +517,31 @@ def timed(
     return (time_total, result) if return_result else time_total
 
 
+def _normalize_bench_inputs(example_inputs) -> Tuple[Tuple[Any], Mapping[str, Any]]:
+    # NOTE(bowbao): For huggingface benchmark, example_inputs are formatted as dictionary,
+    # and consumed like `model(**example_inputs)`.
+    # For other benchmarks, example_inputs are formatted as tuple and consumed
+    # like `model(*example_inputs)`.
+    if isinstance(example_inputs, dict):
+        return (), example_inputs
+    else:
+        return example_inputs, {}
+
+
+def _register_dataclass_output_as_pytree(example_outputs) -> None:
```

Comment:
This is ok for benchmarking, but we need to think about how this should work in a non-python environment. cc @chenyang78